Pod/Classes/URBNAlertView.m

```diff
@@ -82,11 +82,19 @@ - (instancetype)initWithAlertConfig:(URBNAlertConfig *)config alertStyler:(URBNA
         
         // Add some buttons
         NSMutableArray *btns = [NSMutableArray new];
+        NSMutableArray *separators = [NSMutableArray new];
         buttonContainer.translatesAutoresizingMaskIntoConstraints = NO;
         
         __weak typeof(self) weakSelf = self;
         [self.alertConfig.actions enumerateObjectsUsingBlock:^(URBNAlertAction *action, NSUInteger idx, BOOL *stop) {
             if (action.isButton) {
+                if ([self.alertStyler.separatorHeight boolValue]) {
```

Comment:
Not a fan of this. Converting numbers to bools doesn't feel right. Would be better to check it numerically